.github/workflows/pull-request.yml

```diff
@@ -84,4 +84,15 @@ jobs:
           target: default
           arch: x86_64
           profile: pixel_3a
-          script: ./gradlew connectedCheck
\ No newline at end of file
+          script: ./gradlew connectedCheck
+
+  security:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Run Snyk to check for vulnerabilities
+        uses: snyk/actions/gradle@master
+        env:
+          SNYK_TOKEN: 24e1bfa6-f30e-42fd-b996-a71cec1b0967
```

Comment:
i will change this once platform creates the secret

Reply:
this secret is forever leaked now.. we need to rotate this credential if possible